apps/src/blockly/addons/functionEditor.ts

```diff
@@ -125,6 +125,18 @@ export default class FunctionEditor {
 
     const functionEditorTrashcan = new CdoTrashcan(this.editorWorkspace);
     functionEditorTrashcan.init();
+    // Set primary workspace to be active (until a function is shown).
+    Blockly.common.setMainWorkspace(this.primaryWorkspace);
```

Comment:
why is this needed now?

Reply:
The active ("main") workspace is the one used for keyboard shortcuts, including the one used to turn on keyboard navigation. Without this, if you toggled navigation on immediately upon page load it would actually enable it for the editor workspace since it was the active one. 

We also set the primary workspace to be active when the editor is closed, and do the opposite when we show the editor for a function.